app/representers/api/v1/user_representer.rb

```diff
@@ -193,6 +193,14 @@ class UserRepresenter < Roar::Decorator
                if: ->(user_options:, **) { user_options.try(:fetch, :include_private_data, false) },
                getter: ->(represented:, **) { represented.external_ids.map(&:external_id) }
 
+    property :assignable_school_integrated,
+               if: ->(user_options:, **) { user_options.try(:fetch, :include_private_data, false) },
+               getter: ->(*) { self.school.has_assignable_contacts unless self.school.nil?}
```

Comment:
You can probably do `self.school&.has_assignable_contacts`

Reply:
I worry about adding these things to this representer though, as the representer is used to make the SSO cookie, which runs into size issues often...

Reply:
Heck... most of this stuff probably isn't needed in the SSO cookie...

Reply:
All the inline comments on the files got deleted from the diff when I updated the branch from main... but I'm pretty sure this is the last remaining issue. 
How do I keep this in the API but not bloat the cookie?